common/classify.py

```diff
@@ -94,9 +101,19 @@ def test_and_report(self):
         if not os.path.exists(self.results_dir):
             os.makedirs(self.results_dir) 
 
+        logging_interval = 60 # TODO(matthew): Move to config.yaml
+        progress_logger.info("Starting to test on data.")
+        start_time = time.time()
+        last_logged_time = start_time
+
         results = np.empty(shape=(0, TOTAL_SIZE), dtype='S200')
         
         for root, dirs, files in os.walk(self.email_path):
+            curr_time = time.time()
+            if (curr_time - last_logged_time) > logging_interval * 60:
+                progress_logger.info('Exploring directory: {}'.format(root))
```

Comment:
2. A reasonable way would be to truncate the directory name so it has just the first 3 characters after `broScripts/output/` and truncate the rest.